js/researches/getSentenceBeginnings.js

```diff
@@ -0,0 +1,43 @@
+var getSentences = require( "../stringProcessing/getSentences.js" );
+var getWords = require( "../stringProcessing/getWords.js" );
+var stripSpaces = require( "../stringProcessing/stripSpaces.js" );
+var firstWordExceptions = require ( "../config/firstWordExceptions.js" )();
+
+/**
+ * Compares the first word of each sentence with the first word of the following sentence.
+ * @param {array} sentenceBeginnings The array containing the first word of each sentence.
+ * @returns {array} The array containing the objects containing the first words and the corresponding counts.
+ */
+var compareFirstWords = function ( sentenceBeginnings ) {
```

Comment:
It feels like this method is doing more than just one task.
It's currently both checking whether the first word in the sentence matches that of the following sentence _and_ pushes a value to the counts array.

You might want to split up the matching of the two sentences into a separate method which returns true or false.
Based on that you could manipulate the `count` or the `counts` array.